### DISCUSSION TITLE
【项目启动报错】就重新安装了依赖结果就启动不了

### DISCUSSION THREAD

**Discussion by lkzwc** [umijs/umi#11302] [Q&A]
![image](https://github.com/umijs/umi/assets/84896877/4df17c53-9d07-4702-8b09-e6101872c859)
项目启动之后f访问界面报错 ，启动时OK的，但是进入界面的时候就直接报错TypeError: pathname.match is not a function

**Comment by xiaohuoni** [comment]
看 lock 文件什么包更新了版本
这里应该是 pathname 空了

**Comment by fz6m** [comment]
mfsu 关了试试？

umi 3 不维护了，建议升级到 umi 4

**Comment by dingchuancao** [comment]
解决了吗

**Comment by Ccc-boop-eng** [comment]
我也遇到这个问题了，求解
![image](https://github.com/user-attachments/assets/664eba79-3558-440b-a2e7-ad23bd725060)
